.drone.yml

```diff
@@ -43,6 +43,7 @@ steps:
         path: /var/lib/pacman/sync
     environment:
       PKGDEST: /drone/src/out
+      MAKEFLAGS: -j4
```

Comment:
I would have preferred this (and the change n `tools/`) to be in a separate PR. Oh well.